bots/pr/src/test/java/org/openjdk/skara/bots/pr/PullRequestCommandTests.java

```diff
@@ -357,4 +359,40 @@ void summaryCommandInBodyWithBotComment(TestInfo testInfo) throws IOException {
                                           "```\n");
         }
     }
+
+    @Test
+    void interpretCommandFromReviewsAndReviewComments(TestInfo testInfo) throws IOException {
+        try (var credentials = new HostCredentials(testInfo);
+             var tempFolder = new TemporaryDirectory()) {
+            var author = credentials.getHostedRepository();
+            var integrator = credentials.getHostedRepository();
+
+            var censusBuilder = credentials.getCensusBuilder()
+                    .addAuthor(author.forge().currentUser().id());
+            var prBot = PullRequestBot.newBuilder().repo(integrator).censusRepo(censusBuilder.build()).build();
+
+            // Populate the projects repository
+            var localRepo = CheckableRepository.init(tempFolder.path(), author.repositoryType());
+            var masterHash = localRepo.resolve("master").orElseThrow();
+            assertFalse(CheckableRepository.hasBeenEdited(localRepo));
+            localRepo.push(masterHash, author.url(), "master", true);
+
+            // Make a change with a corresponding PR
+            var editHash = CheckableRepository.appendAndCommit(localRepo);
+            localRepo.push(editHash, author.url(), "edit", true);
+            var pr = credentials.createPullRequest(author, "master", "edit", "This is a pull request");
+
+            // Test command in review
+            pr.addReview(Review.Verdict.APPROVED, "/reviewers 3");
+            // Run the bot
+            TestBotRunner.runPeriodicItems(prBot);
+            assertLastCommentContains(pr, "The total number of required reviews for this PR (including the jcheck configuration and the last /reviewers command) is now set to 3");
+
+            //Test command in reviewComment
+//            pr.addReviewComment(masterHash, editHash, Path.of("appendable.txt").toString(), 2, "/reviewers 4");
+//            // Run the bot
+//            TestBotRunner.runPeriodicItems(prBot);
+//            assertLastCommentContains(pr, "The total number of required reviews for this PR (including the jcheck configuration and the last /reviewers command) is now set to 4");
```

Comment:
Ditto the recommendation to remove this.

Reply:
Sure